pkg/converter/convert_unix.go

```diff
@@ -589,6 +589,18 @@ func Merge(ctx context.Context, layers []Layer, dest io.Writer, opt MergeOption)
 				}
 				defer bootstrap.Close()
 
+				// Check if the bootstrap is already generated
+				bootCacheName := filepath.Join(opt.WorkDir, layers[idx].Digest.Hex())
```

Comment:
My suggestion is that external callers provide `layer.ReaderAt` implementations, rather than replacing bootstrap here.

Reply:
fixed.

Reply:
fixed